Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
Re naming: Happy to go with `xv`, but I would mention that `xv` and `lzma` really refer to two different things (a tool and a compression method). It is unlikely libtiff would "catch up" as its options are compression methods.